src/unix/mod.rs

```diff
@@ -204,6 +218,22 @@ pub const PRIO_MIN: ::c_int = -20;
 pub const PRIO_MAX: ::c_int = 20;
 
 cfg_if! {
+    if #[cfg(target_os = "openbsd")] {
+        // P_* constants are not available
+    } else if #[cfg(target_os = "freebsd")] {
```

Comment:
Similar to above, could this `cfg_if!` be avoided to push the definitions into lower modules? (duplication is fine)